src/Benchmarks/Data/DapperDb.cs

```diff
@@ -58,6 +58,28 @@ public DapperDb(IRandom random, DbProviderFactory dbProviderFactory, IOptions<Ap
             return result;
         }
 
+        public async Task<World[]> LoadMultipleUpdatesRows(int count)
+        {
+            var result = await LoadMultipleQueriesRows(count);
+
+            using (var db = _dbProviderFactory.CreateConnection())
+            {
+                db.ConnectionString = _connectionString;
+                await db.OpenAsync();
+
+                for (int i = 0; i < count; i++)
+                {
+                    var randomNumber = _random.Next(1, 10001);
+                    await db.ExecuteAsync("UPDATE world SET randomNumber = @Random WHERE id = @Id", new { Random = randomNumber, Id = result[i].Id });
+                    result[i].RandomNumber = randomNumber;
+                }
+
+                db.Close();
```

Comment:
I think `db.Close()` is unnecessary, since it will be disposed by the `using` statement.